db/re/item_db_equip.yml

```diff
@@ -43827,7 +43825,7 @@ Body:
       NoAuction: true
     Script: |
       bonus2 bSubEle,Ele_Fire,10;
-      bonus4 bAutoSpellWhenHit,"MG_FIREBALL",5,500,1;
+      bonus4 bAutoSpellWhenHit,"MG_FIREBALL",5,30,1;
```

Comment:
Maybe our data is outdated but it's 500 in our aegis file, please revert
```suggestion
      bonus4 bAutoSpellWhenHit,"MG_FIREBALL",5,500,1;
```

Reply:
Thanks